modules/auxiliary/admin/http/joomla_registration_privesc.rb

```diff
@@ -58,7 +58,11 @@ def check
     version = Gem::Version.new(joomla_version)
     if version
       print_status("Detected Joomla version #{joomla_version}")
-      return Exploit::CheckCode::Appears if version.between?(Gem::Version.new('3.4.4'), Gem::Version.new('3.6.3'))
+      if version.between?(Gem::Version.new('3.4.4'), Gem::Version.new('3.6.3'))
+        return Exploit::CheckCode::Appears
+      else
+        return Exploit::CheckCode::Safe
+      end
     end
 
     return Exploit::CheckCode::Detected if online
```

Comment:
I know it's not part of the changes in this PR, but this happy little conditional `if online`  should always be `true`, by virtue of bailing out earlier with `unless online`, rendering it pointless.

```suggestion
    return Exploit::CheckCode::Detected
```

This also makes allocating `online` as a variable a few lines above totally pointless too.